code/src/components/EmptyState.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import styled from "styled-components";
+
+export const EmptyState = () => {

```

Comment:
Nice to have an empty state component. You're getting the hang of the beauty of React and components now ☺️